db-scripts/src/main/resources/adjust_col_size_to_utf8.sql

```diff
@@ -0,0 +1,16 @@
+DROP PROCEDURE IF EXISTS adjust_col_size_to_utf8;
```

Comment:
@khzhu So a while ago I thought to introduce a stored procedure to cleanup orphan records in certain tables but @ersinciftci made a valid point that doing so may lead to problems trying to debug issues - something along those lines. 

@ersinciftci can you comment on this?

Reply:
@khzhu I also recently introduced a PR that changed all allele columns in `mutation` and `mutation_event` to `TEXT` datatypes and we are instead doing a partial index on `mutation_event.TUMOR_SEQ_ALLELE`.

Reply:
with your PR, this no longer needed, will remove.